SU-SWS/minimally_branded_subtheme
# D8CORE-5779: fixup

# READY FOR REVIEW

# Summary
- Fixes to minimally branded subthemed=

# Review By (Date)
- 4/28

# Criticality
- Normal

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Import content from `amptesting-dev`
4. Navigate to ticket to check list of items.
5. Verify they are fixed per design request.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? NO

## Front End Validation

## Backend / Functional Validation
### Code

### Code security

## General

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [D8CORE-5779](https://stanfordits.atlassian.net/browse/D8CORE-5779)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)
